source/reference/api/start.txt

```diff
@@ -111,6 +120,61 @@ Request Body Parameters
 
        Default value is ``false``.
 
+
+.. _c2c-api-start-sharding:
+
+Sharding Parameters
```

Comment:
[question]

Would "fields" be a more appropriate name than "parameters" for these options? My understanding is that these are subfields of the `sharding` object.

Reply:
Fixed

Reply:
Acknowledged. In that case, I'm fine leaving this as-is. Thanks for clarifying.